README.md

```diff
@@ -117,6 +117,20 @@ require'treesitter-context'.setup{
     -- Separator between context and content. Should be a single character string, like '-'.
     -- When separator is set, the context will only show up when there are at least 2 lines above cursorline.
     separator = nil,
+    -- A pre-hook that allow to disable treesitter-context based on buffer options
+    -- Here is a example where treesitter-context is disable for json and python files
+    -- Other custom logic can be assigned here related to buffer
+    -- If condition is true then treesitter-context will disable
+    condition = function(buf)
+      local ft = vim.fn.getbufvar(buf, "&filetype")
```

Comment:
You could write this using the nvim api:
```suggestion
      local ft = vim.api.nvim_buf_get_option(buf, "filetype")
```

Reply:
Thanks for the suggestion. Yes api is always better.